server/apps/duplication/archive_fetch.py

```diff
@@ -52,23 +52,24 @@ class FetchResource(Resource):
 class FetchService(BaseService):
 
     def create(self, docs, **kwargs):
-        return self.__fetch(docs, guid=request.view_args.get('guid'), **kwargs)
+        return self.__fetch(docs, id=request.view_args.get('id'), **kwargs)
 
-    def __fetch(self, docs, guid=None, **kwargs):
-        guid_of_fetched_items = []
+    def __fetch(self, docs, id=None, **kwargs):
+        id_of_fetched_items = []
 
         for doc in docs:
-            guid_of_item_to_be_fetched = doc['guid'] if guid is None else guid
+            id_of_item_to_be_fetched = doc.get('_id') if id is None else id
```

Comment:
Agreed, but that's the syntax of service handler in `BaseService` for `POST` request